afctl/parsers.py

```diff
@@ -257,11 +258,17 @@ def act_on_configs(cls, args, config_file):
             # Setting global values.
             if args.type == "global":
                 origin = args.o
-                if args.o is None:
+                token = args.t
+                if args.o is None and args.t is None:
                     origin = input("Git origin for deployment : ")
-                if origin != '':
+                    token = input("Personal access token : ")
+
+                if origin != '' and origin is not None:
                     Utility.update_config(config_file, {'global':{'git':{'origin':origin}}})
 
+                if token != '' and token is not None:
+                    Utility.update_config(config_file, {'global':{'git':{'access-token':token}}})
```

Comment:
we are struggling in qds sdk because of setting class variables, we are unable to maintain multiple sessions as a result. Can you explain the use case here? Can we do something better?

Reply:
Please elucidate. Unable to understand.
There are no class variables used here.